clients/restclient/rest_client.go

```diff
@@ -43,3 +50,23 @@ func (c RestClient) PurgeConfiguration(org, space string) error {
 	}
 	return nil
 }
+
+func (c RestClient) getCsrfToken() error {
```

Comment:
If you want this to be callable from outside the package, you need to declare it with the capital letter: GetCsrfToken :)